apps/OpenSpace/main.cpp

```diff
@@ -1117,22 +1117,17 @@ int main(int argc, char* argv[]) {
         }
         LINFO(fmt::format("Configuration Path: '{}'", configurationFilePath));
 
+        // Register the base path as the directory where 'filename' lives
```

Comment:
Was a bit confused by this comment at first. Maybe change to "Register the base path as the directory where the configuration file lives"

Reply:
Good point.  That function was just verbatim lifted out of the `loadConfigurationFromFile` function without renaming it